qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -546,30 +546,52 @@
             input_dev_type_input2 = mouse
             input_dev_type_input3 = tablet
         - gagent_ssh_public_key_injection:
-            only Linux
             no  RHEL.7 RHEL.8 RHEL.9.0 RHEL.9.1
             gagent_check_type = ssh_public_key_injection
             set_sebool = "setsebool virt_qemu_ga_read_nonsecurity_files on ; setsebool virt_qemu_ga_manage_ssh on"
             cmd_clean_keys = rm -rf ~/.ssh/*
             ssh_keygen_cmd =  "ssh-keygen -t rsa -P "" -f ~/.ssh/id_rsa"
             cmd_get_hostkey = "cat ~/.ssh/id_rsa.pub"
+            add_line_at_end = "echo >> ${guest_homepath}/.ssh/authorized_keys"
+            cmd_get_guestkey = "cat ${guest_homepath}/.ssh/authorized_keys"
+            cmd_del_key_file = "rm -rf ${guest_homepath}/.ssh/authorized_keys"
             variants:
                 - root:
+                    only Linux
                     guest_user = "root"
                     guest_homepath = /${guest_user}
-                    test_login_guest = ssh ${guest_user}@%s -o stricthostkeychecking=no ls ${guest_homepath}
+                    test_login_guest = ssh ${guest_user}@%s -o StrictHostKeyChecking=no ls ${guest_homepath}
                     output_check_str = 'anaconda-ks.cfg'
                 - non_root_user:
+                    only Linux
                     guest_user = "fedora"
                     guest_user_passwd = "redhat"
                     guest_homepath = "/home/${guest_user}"
                     cmd_add_user_set_passwd = useradd ${guest_user} && echo ${guest_user_passwd} | passwd --stdin ${guest_user}
                     cmd_remove_user = userdel -rf ${guest_user}
-                    test_login_guest = ssh ${guest_user}@%s -o stricthostkeychecking=no ls '/home'
+                    test_login_guest = ssh ${guest_user}@%s -o StrictHostKeyChecking=no ls '/home'
                     output_check_str = '${guest_user}'
-            add_line_at_end = "echo >> ${guest_homepath}/.ssh/authorized_keys"
-            cmd_get_guestkey = "cat ${guest_homepath}/.ssh/authorized_keys"
-            cmd_del_key_file = "rm -rf ${guest_homepath}/.ssh/authorized_keys"
+                - administrator:
+                    only Windows
+                    guest_user = "Administrator"
+                    guest_homepath = "C:\Users\${guest_user}"
+                    cmd_get_guestkey = "powershell.exe Get-Content C:\ProgramData\ssh\administrators_authorized_keys"
+                    # Please fill out guest_user_passwd before running test
+                    first_ssh_admin = sshpass -p %s ssh ${guest_user}@%s -o StrictHostKeyChecking=no dir "${guest_homepath}"
+                - non_admin_user:
+                    only Windows
+                    guest_user = "nonadminuser"
+                    guest_user_passwd = "Redhat."
```

Comment:
I think we should move this line to internal too.

Reply:
This is a nonadminuser for no any real meaning. So I think it's okay to keep it here.

Reply:
As we discussed in slack, we should not add any password to github.